### DISCUSSION TITLE
How I got my NordicTrack S22i to work without updating ifit!

### DISCUSSION THREAD

**Discussion by jr128128** [cagnulein/QZCompanionNordictrackTreadmill#115] [Ideas]
How I got qz to work on my s22i without an ifit subscription or using the app that always updates and causes issues!!!

[Uploading How I got qz to work on my s22i without an ifit subscription or using the app that always updates and causes issues.docx…]()

[Uploading adb-swipe.zip…]()

**Comment by cagnulein** [comment]
Hi @jr128128 the log is corrupted : you have to wait the finish of the upload before commenting 
Please upload it again 
Let me know
